kubetest2/internal/deployers/eksapi/deployer.go

```diff
@@ -263,6 +263,10 @@ func (d *deployer) verifyUpFlags() error {
 	if d.NodeReadyTimeout == 0 {
 		d.NodeReadyTimeout = time.Minute * 5
 	}
+	if d.AMIType == "" {
```

Comment:
LGTM, just double check, we won't check against the version to defaulting?

Reply:
good point, ill only add the field to the request if the AMIType is provided and non-empty so that it just copies MNG behavior rather than specifying our own